teammapper-backend/src/map/entities/mmpMap.entity.ts

```diff
@@ -36,4 +36,7 @@ export class MmpMap {
   })
   /* eslint-enable @typescript-eslint/no-unused-vars */
   nodes: MmpNode[]
+
+  @Column({ type: 'timestamptz', default: () => 'now()' })
```

Comment:
the default ist not really fitting here. if possible use the modifiedAt as default or leave it null

Reply:
I've removed the default here entirely, though I'm guessing that we'll create new maps using `createdAt: new Date()`, so I'm not sure how much of a practical difference that'll make.

Reply:
my primarily concern would be existing maps, where this field should be empty. I think adding a new column with a default value would populate also existing old entries which is not wanted.

Reply:
Added defaults now in a secondary migration, so existing maps get a null entry in createdAt.